Jenkinsfile

```diff
@@ -331,6 +330,19 @@ pipeline {
                   script {
                     currentBuild.description = "${currentBuild.description?.trim() ? currentBuild.description : ''} released"
                   }
+                  archiveArtifacts(allowEmptyArchive: true, artifacts: "${env.BASE_DIR}/packages/rum/dist/bundles/*.js")
+                }
+              }
+            }
+            stage('Publish CDN') {
+              options { skipDefaultCheckout() }
+              steps {
+                script {
+                  def version = sh(label: 'Get package version', script: 'jq --raw-output .version packages/rum/package.json', returnStdout: true)
+                  publishToCDN(header: "Cache-Control:public,max-age=3600",
+                              source: 'packages/rum/dist/bundles/*.js',
+                              target: "gs://beats-ci-temp/rum/${version}", // TODO: change bucket
+                              secret: 'secret/observability-team/ci/service-account/test-google-storage-plugin') // TODO: change secret
```

Comment:
Can I query the version for the given package after the release? Or it will be based on the upcoming release?

Reply:
After the release, sounds good.